### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by andreybolonin**
add symfony5 compatibility

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
   "require": {
     "php": ">=7.1",
     "facebook/webdriver": "^1.7.1",
-    "symfony/browser-kit": "^4.3",
-    "symfony/dom-crawler": "^4.3",
-    "symfony/http-client": "^4.3",
+    "symfony/browser-kit": "^4.3|^5.0",
+    "symfony/dom-crawler": "^4.3|^5.0",
+    "symfony/http-client": "^4.3|^5.0",
```
Comment (nicolas-grekas) [inline composer.json]: It would be consistent to use the same style as below: double pipe+spaces around.

**Comment by manzolo** [comment]
To pass tests it is necessary to solve this problem https://github.com/symfony/panther/issues/245

**Comment by webignition** [comment]
PR #248 fixes #245 and  #246 which are both needed to be resolved for this build to pass

**Comment by dunglas** [comment]
Thanks @andreybolonin

**Comment by andreybolonin** [comment]
@dunglas thx

**Pull request merged**